README.md

```diff
@@ -1,3 +1,3 @@
 # G Suite Solutions Gallery
 
-Welcome to the repository for G Suite solutions.
+Welcome to the repository for G Suite solutions where you can browse end to end use cases for you business needs. Try one out today! Feeling inspired? Browse our [developer documentation](https://developers.google.com/gsuite/) to get started.
```

Comment:
I'm not Tech Writer, but I think there should be a comma before the "where you can". Also "you business needs" should be "your business needs". Also please format for line length.